### ISSUE TITLE
Fix batch numbering in training logs

### ISSUE THREAD

**Issue by nbren12** [VulcanClimateModeling/fv3net#1200]
Currently, the ML training logs use zero-based indexing for the batch numbering, leading to a confusing statement like this:
```
INFO:SklearnWrapper:Fitting batch 0/1
```

this should be modified to 
```
 INFO:SklearnWrapper:Fitting batch 1/1
```

**Comment by nbren12** [comment]
@AnnaKwa @spencerkclark

**Comment by oliverwm1** [comment]
This was closed by #1214
